src/flask/helpers.py

```diff
@@ -666,12 +668,24 @@ def download_file(name):
         ``directory``.
     :param kwargs: Arguments to pass to :func:`send_file`.
 
+    .. versionchanged:: 2.0
+        ``path`` replaces the ``filename`` parameter.
+
     .. versionadded:: 2.0
         Moved the implementation to Werkzeug. This is now a wrapper to
         pass some Flask-specific arguments.
 
     .. versionadded:: 0.5
     """
+    if filename is not None:
+        warnings.warn(
+            "The 'filename' parameter has been renamed to"
+            " 'path'. The old name will be removed in Flask"
+            " 2.1.",
+            DeprecationWarning,
+            stacklevel=3,
```

Comment:
Was `stacklevel=3` intentional? Seems like it should be `2`.

Reply:
I changed it to 2.

Reply:
In the beginning, I tried to write a wrapper function like the one for `send_file`, then find it is unnecessary, so I remove the wrapper, but forgot to update the `stacklevel`.